internal/clients/nightfall/nightfall.go

```diff
@@ -175,6 +183,34 @@ func sliceListBySize(index, numItemsForMaxSize int, contentToScanList []*content
 	return contentToScanList[startIndex:endIndex]
 }
 
+func createCommentsFromScanRespForFiles(inputContent []*fileToScan, resp *nf.ScanTextResponse, tokenExclusionList []string) []*diffreviewer.Comment {
+	comments := make([]*diffreviewer.Comment, 0)
+	for j, findingList := range resp.Findings {
+		for _, finding := range findingList {
+			if finding.Finding != "" && !isFindingInTokenExclusionList(finding.Finding, tokenExclusionList) {
+				// Found sensitive info
+				// Create comment if fragment is not in exclusion set
+				correspondingContent := inputContent[j]
+				exists, lineNumber, _ := correspondingContent.ContentToLineMap.Find(int(finding.Location.ByteRange.Start))
```

Comment:
I think here we should use `finding.Location.CodePointRange.Start` as everywhere else we are counting the number of chars not bytes.